### PR TITLE
fixes #239

### LINKED ISSUE #239: change "Save" and "publish' buttons in the uploader
Basically, the idea is this:

* if the video is not pbulsihed, show "publish" instead of "save data". Remove the existing "cancel" and "publish" buttons in the bottomo right. Clicking on this button should (a) save the data and (b) show the staking modal
* one ce the vid is published: show "save" one.

### PR DESCRIPTION
**Pull request by milankragujevic**
my first commit to portal 👍

### CODE REVIEW

**Comment by milankragujevic** [comment]
sorry i acidentaly committed the wrong thing (deleted the right file), ignore this please

**Comment by bent0b0x** [comment]
@milankragujevic should we close this? or will you be pushing an update

**Comment by milankragujevic** [comment]
@bent0b0x Im recreating the commit as we speak, so yeah Ill push more code very soon...

File: src/scripts/components/VideoForm.js [hunk 1]
```diff
@@ -259,17 +263,28 @@ class VideoForm extends Component<Props, Object> {
...
+                  onClick={e => {
+                    this.handleSubmit(e)
+                    this.props.showModal(
+                      <ModalStake
+                        videoId={this.state.id}
+                        closeModal={this.props.closeModal}
+                      />
+                    )
+                  }}
+                  // disabled={!this.props.canSubmit}
```
Comment (bent0b0x) [inline src/scripts/components/VideoForm.js]: Can you remove this comment?

Comment (milankragujevic) [reply src/scripts/components/VideoForm.js]: ok

**Pull request closed without merging**